packages/vertica-nodejs/test/integration/client/simple-query-tests.js

```diff
@@ -8,9 +8,19 @@ types.setTypeParser(VerticaType.Integer, function(val) {
   return parseInt(val, 10)
 })
 
+function tempSetup(client) {
```

Comment:
Were the tests previously depending on tables created elsewhere?

Reply:
Creation and cleanup of test data is a good place to use Mocha before/after or beforeEach/afterEach.

Although I see the vertica-nodejs tests are implemented differently from the other modules which probably complicates things.